### ISSUE TITLE
 java.lang.ClassCastException: Must be a class java.lang.Boolean not a class com.google.common.base.Absent

### ISSUE THREAD

**Issue by L4BORG** [TechzoneMC/SonarPet#65]
```
[17:53:56 WARN]: Entity threw exception at world:-777.5414954810243,65.0,2825.554086912626
[17:53:56 WARN]: java.lang.ClassCastException: Must be a class java.lang.Boolean not a class com.google.common.base.Absent
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.metadata.MetadataType.checkCast(MetadataType.java:41)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.metadata.MetadataKey.cast(MetadataKey.java:23)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.metadata.WrappedDataWatcher.get(WrappedDataWatcher.java:23)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.entity.type.EntityEndermanPet.isScreaming(EntityEndermanPet.java:56)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.entity.type.EntityEndermanPet.getIdleSound(EntityEndermanPet.java:61)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.entity.EntityInsentientPet.G(EntityInsentientPet.java:469)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.entity.type.EntityEndermanPet.G(EntityEndermanPet.java:127)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.EntityInsentient.D(EntityInsentient.java:172)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.EntityInsentient.U(EntityInsentient.java:185)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.Entity.m(Entity.java:321)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.EntityLiving.m(EntityLiving.java:1766)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.EntityInsentient.m(EntityInsentient.java:241)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.EntityMonster.m(EntityMonster.java:32)
[17:53:56 WARN]:        at com.dsh105.echopet.compat.nms.v1_10_R1.entity.type.EntityEndermanPet.m(EntityEndermanPet.java:88)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.World.entityJoinedWorld(World.java:1724)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.World.g(World.java:1699)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.World.tickEntities(World.java:1523)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.WorldServer.tickEntities(WorldServer.java:635)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:865)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:403)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:730)
[17:53:56 WARN]:        at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:629)
[17:53:56 WARN]:        at java.lang.Thread.run(Thread.java:745)
```

latest dev on latest TacoSpigot 1.10

**Comment by Techcable** [comment]
WOW mojang, apparently whether an enderman is screaming is ` Optional<Boolean>`.

**Comment by Techcable** [comment]
Should be fixed as of 772727029c2b64eb79c6ccbeee396bcd148e8795
